src/components/ModalPortal.js

```diff
@@ -71,6 +71,15 @@ export default class ModalPortal extends Component {
 
     this.shouldClose = null;
     this.moveFromContentToOverlay = null;
+
+    // Body focus trap see Issue #742
+    this.bodyFocusTrapBefore = document.createElement("div");
+    this.bodyFocusTrapBefore.style.position = "absolute";
+    this.bodyFocusTrapBefore.style.opacity = "0";
+    this.bodyFocusTrapBefore.setAttribute("tabindex", "0");
+    this.bodyFocusTrapAfter = this.bodyFocusTrapBefore.cloneNode();
+    this.bodyFocusTrapBefore.addEventListener("focus", this.focusContent);
+    this.bodyFocusTrapAfter.addEventListener("focus", this.focusContent);
```

Comment:
This will create the trap elements for each modal created. 

Perhaps you can move this code to a new file and create these elements
in a singleton fashion.

Another question, though, is what would happen if there are nested modals opened?

Reply:
Maybe we need to track the lastest modal open (?)


NOTES: 
Probably we need to rewrite the way we manage nested modals.

Reply:
@diasbruno Yeah nested modals was why I was thinking it might be easier to just make a set of these per-modal. They 'wrap' the `<body>` nodes, and each subsequent modal adds another wrapping layer that's associated with only a single modal so giving focus via `this.bodyFocusTrapBefore.addEventListener("focus", this.focusContent);` is easy.

Happy to change it to a singleton approach though (that would make fewer `<div/>`s at the expense of a bit more complexity)... you mentioned a new file so I guess in that file I'd have a array of ModalPortal instances... pushing new ones on in `beforeOpen` and popping off in `afterClose`, and calling `this.focusContent` on the top modal when these trap elements were focused. Is that what you thinking?

Reply:
> I'd have a array of ModalPortal instances... pushing new ones on in beforeOpen and popping off in afterClose, and calling this.focusContent on the top modal when these trap elements were focused.

It seems the right way to go.

Reply:
@diasbruno is this what you were thinking, or is this too generic?

Reply:
It's great.